cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -422,19 +422,19 @@ private ApiResponse createUser(List<String> subAccountsIds, Boolean enabled) thr
     }
 
     private ApiResponse createUser(List<String> subAccountsIds, Account.Role role) throws Exception {
-        String email = String.format("%s@%s.com", randomLetters(), randomLetters());
+        String email = String.format("sdk+" + SDK_TEST_TAG + randomLetters() + "@cloudinary.com");
```

Comment:
@adimiz1 can't we get the travis job ID (if running in the CI of course) ?

Reply:
@asisayag2 it's part of the SDK_TEST_TAG parameter
in case you wonder about the random letters it's because each job runs 3 different create user tests